tools/pipelines/templates/build-npm-client-package.yml

```diff
@@ -429,8 +429,6 @@ extends:
 
                       - template: /tools/pipelines/templates/include-telemetry-setup.yml@self
                         parameters:
-                          devFeedUrl: $(ado-feeds-dev)
-                          officeFeedUrl: $(ado-feeds-office)
```

Comment:
Passing these as parameters to the template every time is not necessary anymore. The bottom-level template now tries to use them directly and validates that they are not empty. They still need to be provided at the top level by including the `ado-feeds` variable group, but we don't need to pass them explicitly down the template chain.